README.md

```diff
@@ -23,13 +25,18 @@ git init
 ```
 5. Change the package.json file to information about your project
 6. Delete this README and create a new one for your project
-7. Commit all the files as an initial commit and push to your origin
+7. Create a new branch titled `initial-setup`
+```
+git checkout -b initial-setup
+```
+8. Commit all the files as an initial commit and push to your origin
```

Comment:
So are we saying here that we want the initial commit to be in a branch for review rather than making an initial commit on `master` and pushing it up?

If that's the case then do you think we might need a step about adding the GitHub repo as the remote origin?

Reply:
To your first question: yes. I think the reviewer will mostly be skimming over everything, but since it's setting up the foundation for the project, I think it'd be good to have a second set of eyes to make sure it was all done to our standards.

To your second question: Isn't that line 34?

Reply:
Ha, you're right about line 34 🤦‍♂️ 

I didn't look at the stuff not in green lol